src/include/daos_srv/pool.h

```diff
@@ -190,4 +190,7 @@ map_ranks_init(const struct pool_map *map, enum map_ranks_class class,
 
 void
 map_ranks_fini(d_rank_list_t *ranks);
+
+int ds_pool_get_ranks_by_uuid(const uuid_t pool_uuid, int status,
```

Comment:
_by_uuid() is unnecessary, this is a pretty clear API

Reply:
sorry, I accidentally pushed 2 patches in this request. ok, I fixed this in https://github.com/daos-stack/daos/pull/908